<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>About Me</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">Resume</a>
</li>
<li>
  <a href="dashboard.html">Dashboard</a>
</li>
<li>
  <a href="mailto:&lt;yd2868@cumc.columbia.edu&gt;">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/athul12498/athul12498.github.io/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">About Me</h1>

</div>


<div id="education" class="section level2">
<h2>Education</h2>
<p><strong>Wuhan University</strong>, Wuhan, China<br />
<em>B.Sc. in Biological Science, Sep 2020 – Jul 2024</em><br />
Cumulative GPA: 3.49 / 4.0</p>
<ul>
<li>Mathematics: Advanced Mathematics C1 &amp; E2, Probability &amp;
Mathematical Statistics B, Linear Algebra B<br />
</li>
<li>Programming: Python, C Programming Language<br />
</li>
<li>GRE: 326 (Verbal 156, Quantitative 170)</li>
</ul>
<hr />
</div>
<div id="research-experience" class="section level2">
<h2>Research Experience</h2>
<div
id="gene-expression-data-analysis-using-mixed-effects-models-aug-2024-present"
class="section level3">
<h3>Gene Expression Data Analysis Using Mixed Effects Models (Aug 2024 –
Present)</h3>
<ul>
<li>Applied mixed-effects models to analyze large-scale gene expression
data (1,176 genes, two conditions).<br />
</li>
<li>Processed microarray data and validated findings using permutation
tests and Gene Set Enrichment Analysis.<br />
</li>
<li>Categorized functionally related genes based on similarity of
expression profiles.</li>
</ul>
</div>
<div
id="gene-expression-profiling-based-on-high-throughput-sequencing-technology-aug-2024-present"
class="section level3">
<h3>Gene Expression Profiling Based on High-Throughput Sequencing
Technology (Aug 2024 – Present)</h3>
<ul>
<li>Used high-throughput sequencing to compare PBMC expression profiles
between tuberculosis patients and controls.<br />
</li>
<li>Conducted RNA extraction, cDNA library preparation, bridge PCR
amplification, and Illumina deep sequencing.<br />
</li>
<li>Identified diagnostic marker genes through differential expression
analysis.</li>
</ul>
</div>
<div
id="undergraduate-thesis-the-impact-of-gravity-axis-on-mouse-place-cells-sep-2023-apr-2024"
class="section level3">
<h3>Undergraduate Thesis: The Impact of Gravity Axis on Mouse Place
Cells (Sep 2023 – Apr 2024)</h3>
<p><em>Supervised by Prof. Chun Xu, Institute of Neuroscience, Chinese
Academy of Sciences</em><br />
- Recorded hippocampal place cell activity under varying slopes using
GRIN Lens calcium imaging.<br />
- Processed imaging data with CNMF-E and tracked behavior using
DeepLabCut.<br />
- Analyzed discharge data and behavioral correlation to reveal the
effect of gravity on spatial coding.</p>
</div>
<div
id="brain-electrical-activity-acquisition-decoding-algorithms-for-bmis-jul-2023-nov-2023"
class="section level3">
<h3>Brain Electrical Activity Acquisition &amp; Decoding Algorithms for
BMIs (Jul 2023 – Nov 2023)</h3>
<p><em>Published in Highlights in Science, Engineering and Technology,
Vol. 74, 2023</em><br />
- Studied BMI decoding for prosthetic control among amputees and
paralytics.<br />
- Compared invasive vs. noninvasive signal acquisition and examined
linear/nonlinear decoding algorithms.</p>
</div>
<div id="path-academy-online-research-project-jul-2023-nov-2023"
class="section level3">
<h3>Path Academy Online Research Project (Jul 2023 – Nov 2023)</h3>
<p><em>Supervised by Prof. Joni Wallis, UC Berkeley</em><br />
- Led a 4-person team studying the principles and applications of
biological and artificial neural networks.</p>
</div>
<div
id="internship-institute-of-food-science-technology-caas-jul-2022-sep-2022"
class="section level3">
<h3>Internship: Institute of Food Science &amp; Technology, CAAS (Jul
2022 – Sep 2022)</h3>
<p><em>Supervised by Prof. Fengjiao Xin</em><br />
- Searched for phenylalanine ammonia lyase for treating
phenylketonuria.<br />
- Used PyMOL for protein modeling and independently performed protein
purification experiments.</p>
<hr />
</div>
</div>
<div id="leadership-community-involvement" class="section level2">
<h2>Leadership &amp; Community Involvement</h2>
<p><strong>Talent Management Associate</strong>, Wuhan University Center
for Psychological Health Education (Sep 2023 – Jun 2024)<br />
- Organized weekly workshops for 20 student consultants with a 3-member
coordination team.</p>
<p><strong>Vice President</strong>, Martial Art Club, Wuhan University
(Dec 2022 – Jun 2024)<br />
- Recruited new members, maintained club database, and helped develop
club regulations.</p>
<p><strong>Program Counselor</strong>, Wuhan University Common Welfare
Association (Feb 2023 – Jun 2023)<br />
- Co-founded the <em>Luojia Mountain Afterschool Program</em> serving
30+ children from migrant worker families.</p>
<hr />
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><strong>Software:</strong> MATLAB, PyMOL, CaImAn (CNMF-E),
DeepLabCut, Microsoft Office Suite<br />
</li>
<li><strong>Programming:</strong> Python, C++<br />
</li>
<li><strong>Languages:</strong> English (Fluent, TOEFL 101), Chinese
(Native)</li>
</ul>
<hr />
</div>
<div id="publications" class="section level2">
<h2>Publications</h2>
<ol style="list-style-type: decimal">
<li><p><strong>Gene Expression Profiling Based on High-Throughput
Sequencing Technology and Its Application in Disease
Diagnosis</strong><br />
<em>Journal of Prevention, Diagnosis and Management of Human
Diseases</em>, 5(1), 32–44, Jan 2025.<br />
<a
href="https://doi.org/10.55529/jpdmhd.51.32.44">https://doi.org/10.55529/jpdmhd.51.32.44</a></p></li>
<li><p><strong>Gene Expression Data Analysis Based on Mixed Effects
Model</strong><br />
<em>Journal of Computer and Communications</em>, 13, 223–235,
2025.<br />
<a
href="https://doi.org/10.4236/jcc.2025.132014">https://doi.org/10.4236/jcc.2025.132014</a></p></li>
<li><p><strong>Brain Electrical Activity Acquisition &amp; Decoding
Algorithms for Using BMIs</strong><br />
<em>Highlights in Science, Engineering and Technology</em>, 74,
1371–1377, Dec 2023.<br />
<a
href="https://doi.org/10.54097/gy823a26">https://doi.org/10.54097/gy823a26</a></p></li>
</ol>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
